### ISSUE TITLE
Need help, can't deploy app

### ISSUE THREAD

**Issue by RobCRees** [OfficeDev/microsoft-teams-icebreaker-app#76]
### 
Hi there,

I am new to the community, but love the tool.

I'm trying to get this deployed for my org. however I'm stuck :(  I've followed all instructions step by step but get hung up when I have to go into teams app studio to design the app... I keep getting this error:

Error while reading manifest.json: "configurableTabs, staticTabs, bots, connectors, or composeExtensions" is required 

Can someone please help me?  The one part I find challenging is the step by step guide didn't go into detail about what needed to happen in App Studio (i.e. - defining tabs, bots connectors etc)... or I could be missing something.

This is my first time doing something like this and appreciate you pointing me in the right direction.

**Comment by Srivatsan0102** [comment]
Can you please attach a screenshot of your entire manifest file? It will be easier to understand if there is any error there that way.

Also, you can manually change the manifest using a text editor as opposed to using app studio to do the same. Once you do that, check if you have custom app sideloading enabled, and add the bot to a team for it to work.

**Comment by RobCRees** [comment]
I guess my biggest question is.. should I just be downloading the manifest from the site and adding my values to that, vs. trying to build my own app?

ROBERT REES
Senior I.T. Manager
424.465.6197

**Comment by Srivatsan0102** [comment]
It is recommended to use the manifest file and follow the step 3 in the deployment guide, as it shows the required fields needed to sideload the bot to teams. After that, you will have to create the zip package with the manifest, color.png and outline.png as the only files in the zip package and sideload that into the team.

**Comment by RobCRees** [comment]
That did the trick!!!!!

To confirm, I deploy this to individual teams right, not just to the entire org?

ROBERT REES
Senior I.T. Manager
424.465.6197

**Comment by RobCRees** [comment]
Last one…. When I click take a tour, nothing Is occurring, should something be happening?

[cid:image001.png@01D640CE.068A83F0]

ROBERT REES
Senior I.T. Manager
424.465.6197

**Comment by RobCRees** [comment]
This is great… today we got our first icebreaker matches, it’s working.

We did notice one issue though…

I added the icebreaker bot to my HR team in MS Teams… all users except for 1 got the match.

Any ideas why one user didn’t get the chatbot match?  Below is a screenshot of her MS Teams and mine… I see it, she doesn’t.

Thoughts?


[cid:image002.png@01D642FE.78155A40]

ROBERT REES
Senior I.T. Manager
424.465.6197

From: Robert Rees
Sent: Friday, June 12, 2020 3:28 PM
To: OfficeDev/microsoft-teams-icebreaker-app <reply@reply.github.com>; OfficeDev/microsoft-teams-icebreaker-app <microsoft-teams-icebreaker-app@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [OfficeDev/microsoft-teams-icebreaker-app] Need help, can't deploy app (#76)

Last one…. When I click take a tour, nothing Is occurring, should something be happening?

[cid:image003.png@01D642FE.78155A40]


name: ROBERT REES
